content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+
+- **Fixed** Gateway status addresses receiving Service VIPs from outside the cluster.
+
+- **Fixed** `use-waypoint` should be a label, for consistency
+  ([Issue #50572](https://github.com/istio/istio/issues/50572))
+
+- **Fixed** build EDS typed cluster endpoints with domain address.
+  ([Issue #50688](https://github.com/istio/istio/issues/50688))
+
+- **Fixed** a bug where injection template incorrectly evaluates when `InboundTrafficPolicy` is set to "localhost".  ([Issue #50700](https://github.com/istio/istio/issues/50700))
+
+- **Fixed** added server-side keepalive to waypoint HBONE endpoints
+  ([Issue #50737](https://github.com/istio/istio/issues/50737))
+
+- **Fixed** a regression in Istio 1.21.0 causing `VirtualService`s routing to `ExternalName` services to not work when `ENABLE_EXTERNAL_NAME_ALIAS=false` is configured.
+
+- **Fixed** empty prefix match in `HTTPMatchRequest` not rejected by the validating webhook.
+  ([Issue #48534](https://github.com/istio/istio/issues/48534))
+
+- **Fixed** a behavioral change in Istio 1.20 that caused merging of ServiceEntries with the same hostname and port names
+to give unexpected results.
+  ([Issue #50478](https://github.com/istio/istio/issues/50478))
+
+- **Fixed** a bug when a Sidecar is resource is defined with multiple egress listeners with different ports of a Kubernetes service, does not merge the ports correctly. This leads to creating only one Cluster with the first port and second port is ignored.
+
+- **Fixed** an issue causing routes to be overwritten by other virtual services.
+
+- **Removed** `values.cni.privileged` flag from `istio-cni` node agent chart in favor of feature-specific permissions.
+  ([Issue #49004](https://github.com/istio/istio/issues/49004))
+
+- **Removed** the `PILOT_ENABLE_HEADLESS_SERVICE_POD_LISTENERS` feature flag.
+
+- **Removed** the `PILOT_ENABLE_INBOUND_PASSTHROUGH` setting, which has been enabled-by-default for the past 8 releases.
+This feature can now be configured using a new [Inbound Traffic Policy Mode](https://github.com/istio/api/blob/9911a0a6990a18a45ed1b00559156dcc7e836e52/mesh/v1alpha1/config.proto#L203).
+
+## Security
+
+- **Updated** default value of the feature flag `ENABLE_AUTO_ENHANCED_RESOURCE_SCOPING` to true.
+
+- **Added** support for path templating in AuthorizationPolicy, See Envoy URI template [docs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/path/match/uri_template/v3/uri_template_match.proto).  ([Issue #16585](https://github.com/istio/istio/issues/16585))
+
+- **Added** support for customizing timeout when resolving `jwksUri`
+  ([Issue #47328](https://github.com/istio/istio/issues/47328))
+
+- **Added** support for Istio CA to handle node authorization for CSRs with impersonate identity from remote clusters.
+This could help Istio CA to authenticate ztunnel in remote clusters in an external control plane scenario.  ([Issue #47489](https://github.com/istio/istio/issues/47489))
```

Comment:
```suggestion
- **Added** support for Istio CA to handle node authorization for CSRs with impersonating the identity of remote clusters.
  This could help Istio CA to authenticate ztunnel in remote clusters in an external control plane scenario.
    ([Issue #47489](https://github.com/istio/istio/issues/47489))
```